ci/jenkinsfiles/build.groovy

```diff
@@ -54,7 +54,7 @@ pipeline {
     JENKINS_HOME = '/root'
     MAVEN_DEBUG = '-e'
     MAVEN_OPTS = "${MAVEN_OPTS} -Xms512m -Xmx3072m"
-    NUXEO_VERSION = '2021.0'
+    NUXEO_VERSION = '2021.8'
```

Comment:
Don't you want to use the `2021.9` or even the `2021.10` (I think it was released last week)

Reply:
@guirenard is there a reason for us to keep this in `2021.8`?

Reply:
No. It is just at the time I created the branch, it was the latest available platform version